resources/glossary/single-synthetic-environment.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Single Synthetic Environment
+description: "Digital twins are a detailed, simulated analogue to a real-world system"
+slug: single-synthetic-environment
+tags: ["Simulation Modeling", "Databases", "Data Science"]
```

Comment:
Not sure about the _databases_ tag here - would remove it.

More broadly I'm a bit unsure about the _databases_ tag in general. It seems to me that (this aside) the other places we utilize it we could well replace it with "Integrable Systems" or "Integrations", to indicate that these definitions refer to types of software/platforms/systems that can be linked up to HASH.

@nshlapo @ghabs

Reply:
Agree databases shouldn't apply here. If we think of the tags as being focused around how these products/concepts work, integrations is a good  change